engine/common/imagelib/img_dds.c

```diff
@@ -356,7 +362,9 @@ qboolean Image_LoadDDS( const char *name, const byte *buffer, fs_offset_t filesi
 			SetBits( image.flags, IMAGE_HAS_ALPHA );
 		else if( image.type == PF_DXT5 && Image_CheckDXT5Alpha( &header, fin ))
 			SetBits( image.flags, IMAGE_HAS_ALPHA );
-		else if ( image.type == PF_BC7 )
+		else if ( image.type == PF_BC5_SIGNED || image.type == PF_BC5_UNSIGNED )
```

Comment:
No space before parentheses, only inside.

Reply:
Addressed where I could notice it (clang-format when)